compiler/noirc_frontend/src/elaborator/enums.rs

```diff
@@ -518,7 +526,7 @@ impl Elaborator<'_> {
     /// which is an implementation of https://julesjacobs.com/notes/patternmatching/patternmatching.pdf
     pub(super) fn elaborate_match_rows(&mut self, rows: Vec<Row>) -> HirMatch {
         self.compile_rows(rows).unwrap_or_else(|error| {
-            self.push_err(error);
+            self.push_err(error, self.file); // TODO: fix this
```

Comment:
Please make a follow-up issue for this TODO

Reply:
I forgot about this, thanks! It's fixed now.